README.md

```diff
@@ -44,6 +44,15 @@ Screenshots
 
 Changelog
 ------------
+
+#### 1.5.3
+* Added support for requiring shipping and billing information
+* Test against PHP 7.0.10 
```

Comment:
Can we add a line above this stating that something to the effect of `Add support for PHP 7` to help clarify that we support PHP 7?

We should also make a note of the change in behavior where the fields themselves are now marked as required for user input.